x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/create_rule/route.test.ts

```diff
@@ -31,9 +32,7 @@ describe('Create rule route', () => {
 
     clients.rulesClient.find.mockResolvedValue(getEmptyFindResult()); // no current rules
     clients.rulesClient.create.mockResolvedValue(getRuleMock(getQueryRuleParams())); // creation succeeds
-    clients.detectionRulesClient.createCustomRule.mockResolvedValue(
-      getRuleMock(getQueryRuleParams())
-    );
+    clients.detectionRulesClient.createCustomRule.mockResolvedValue(getRulesSchemaMock());
```

Comment:
What was the reason behind this change? Can we keep the existing assertion that used `mockResolvedValue(getRuleMock(getQueryRuleParams()))`?

Reply:
Discussed this during the Zoom call, posting for visibility. 
Here I am changing a mock to `RuleResponse` to match the type return from `createCustomRule`.